docs/pages/eas-update/publish.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: How to publish an update
+sidebar_title: Publish an update
+description: Learn how to publish an update to a specific branch with EAS Update.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { LayersTwo02Icon } from '@expo/styleguide-icons';
+
+Publishing an update allows fixing bugs and quickly updating non-native parts of a project instead of creating a new build. This guide explains the steps required to publish an update, how it works, and methods to test it.
```

Comment:
```suggestion
Publishing an update allows:
- Fixing bugs and quickly updating non-native parts of a project instead of creating a new build
- Sharing a preview version of an app using internal distribution

This guide explains the steps required to publish an update, how it works, and methods to test it.
```